esp/services/ws_config2/ws_config2Service.cpp

```diff
@@ -824,6 +824,18 @@ void Cws_config2Ex::getAttributes(const std::vector<std::shared_ptr<EnvironmentV
             pAttribute->updateType().updateLimits().setChoiceList(choices);
         }
 
+        const std::vector<std::string> &mods = pSchemaValue->getModifiers();
+        if (!mods.empty())
+        {
+            StringArray modifiers;
+            for (auto &modIt: mods)
+            {
+                StringBuffer modifier(modIt.c_str());
```

Comment:
It might be beneficial to declare "modifier" outside of the loop, then modifier.set(modIt.c_str()) on each iteration.